### PR TITLE
Release v3.0.0

### PR DESCRIPTION
**Pull request by rafmudaf**
**Feature or improvement description**
This pull request merges the `dev` branch into the `main` branch for a major version update: v3.0.0. See below for the proposed change log.

**Test results, if applicable**
The baseline results have changed since the last release. See the [r-test pull request](https://github.com/OpenFAST/r-test/pull/44) for more information.

Release checklist:
- [x] Update the documentation version in docs/conf.py
- [x] Update the versions in docs/source/user/api_change.rst
- [x] Verify readthedocs builds correctly
- [ ] Create a tag in OpenFAST
- [ ] Create a merge commit in r-test and add a corresponding tag
- [ ] Compile executables for Windows builds
    - [ ] FAST_SFunc.mexw64
    - [ ] MAP_X64.dll
    - [ ] OpenFAST-Simulink_x64.dll
    - [ ] openfast_x64.exe
    - [ ] DISCON.dll
---

This release includes the following major enhancements:
- FAST.Farm (#584)
- ServoDyn structural control submodule (#607)

### AeroDyn
#597 AD/AA: Add new TE definition, improve airfoil thickness calculation, simplify input
#647 Add ability to turn unsteady aero back on during a simulation
#648 OLAF improvements for WEIS
#672 Preliminary support for multiple rotors in AeroDyn
#728 AllBldOuts more forgiving and support for 0

### BeamDyn
#677 BD Driver bug fix and expand BeamDyn unit testing

### ElastoDyn
#589 Allow one-blade turbine model
#653 Add YawBrTV[xyz]p output channels
#736 AllBldOuts more forgiving and support for 0

### FAST.Farm
#584 Incorporate FAST.Farm to OpenFAST
#726 Build FAST.Farm when configured via BUILD_FASTFARM
#749 Add Missing Attribute for sc_end in the FAST.Farm Super Controller

### HydroDyn
#687 [BugFix] incorrect pitch/roll moments on tapered elements crossing water line
#713 [BugFix] Uninitialized variables in HydroDyn::Morisson calculations

### NWTC Library
#668 [BugFix] Sys files for MATLAB
#683 Big fix in NWTC Library unit tests

### OpenFAST Library
#744 Avoid gfortran 11 error in FAST_Solver.f90

### ServoDyn
#607 ServoDyn Structural control submodule (formerly TMD)
#690 [BugFix] Rename Structural Control driver program
#739 [BugFix] Logic error in glue code for StC loads on SD with no HD

### Documentation
#678 Add documentation on input file parsing
#682 Update installation instructions
#696 Update the api_change for MoorDyn after PR #565

### C++ Inteface
#703 [BugFix] unallocated AD%y%Rotors when AD14 is used with cpp interface
#709 Remove unused import to C++ driver code

### Simulink interface
#702 [BugFix] Correct the expected format of the output channel names array

### Build systems
#698 Configure runtime path linking when using shared libraries
#706 Automate incrementing the dev-label conda build
#725 Updated linking dependencies for newer version of Visual Studio
#727 Update rules for Intel OneAPI compiler detection

### Testing
#670 Enable regression tests for the C++ API
#686 Bug fix for regression tests without C++ API enabled
#689 Enable types generation in CI
#691 Add SubDyn and AeroDyn drivers regression tests to github action
#717 GH Actions: run module tests with debug build
#718 [BugFix] Raise an error if reg test results contain NaN or infinity
#741 CTest: Add FF reg tests when FF build is enabled

### CODE REVIEW

File: docs/source/user/api_change.rst [hunk 1]
```diff
@@ -10,19 +10,33 @@ The line number corresponds to the resulting line number after all changes are i
...
 No changes.
 
 
 
+OpenFAST v2.6.0 to OpenFAST v3.0.0
+----------------------------------
+No changes.
+
+
+
```
Comment (jjonkman) [inline docs/source/user/api_change.rst:24]: The ServoDyn changes below appear to be misplaced.  These are added in v3.0.0, not v2.6.0.

File: docs/source/user/input_file_overview.rst [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+specific line, with some exceptions. 
+
+.. _sec_value_column:
+
+Value column input files
+------------------------
+
+Only the first column in a *value column* based input file is read.  This is the
+historical format used by OpenFAST and it's predecessors. Everything after the
+first value read is simply ignored by the code.  This allowed the user to keep
```
Comment (jjonkman) [inline docs/source/user/input_file_overview.rst:19]: Add something like: "(the keyword was often referenced in the source code and documentation, but OpenFAST did not process the keyword or description)" here.

File: docs/source/user/input_file_overview.rst [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+and which lines contained the value and key pair.  If a keyname is not found the
+parser will return an error with information about which line it was reading
+from.
+
+
+Modules using Key + Value Format
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following modules use the *key + value* format input files (all other
+modules use the *value column* format):
```
Comment (jjonkman) [inline docs/source/user/input_file_overview.rst:75]: Add a note saying something like, "(note that *key + value* format and *value column* input files can be identical if the value is stated before the key)".

File: docs/source/user/input_file_overview.rst [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+   which line the file parsing stopped at. This may help isolate where the input
+   file parsing failed when no line number is given in the error message.
+4. Compare the problematic input file with an input file of the same type from
+   the regression test suite distributed with OpenFAST.  See section
+   :numref:`testing` for details on the regression tests, or check the
+   repository at `r-test <https://github.com/openfast/r-test>`__ .
+
+
+..
+   Input file type by module
```
Comment (jjonkman) [inline docs/source/user/input_file_overview.rst:131]: I'm not sure what this table is for.

**Review by jjonkman — APPROVED** [review]
I'm excited to see this release of OpenFAST v3.0.0.  Please find a few edits and clarifications to the documentation below.

File: docs/source/user/input_file_overview.rst (hunk 4 above)
Comment (andrew-platt) [reply docs/source/user/input_file_overview.rst:131]: This table is commented out and won't render.  I started adding it, but decided to just leave it as a commented out block for later (if we want it).

**Pull request merged**